### ISSUE TITLE
Rework Board Appeals page to match VA.gov style

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#57225]
## Issue Description

We are reworking the decision reviews section to make sure the static pages match the voice and style of VA.gov. These pages were originally put on the site without much collaboration from the Sitewide Content. We need to make sure it meets our plain language standards and follows the same patterns as our other benefit detail pages.

This ticket is for the [Board Appeals](https://www.va.gov/decision-reviews/board-appeal/) page. 

## Tasks
- [ ] Rework page (Katherine)
- [ ] Review (Beth)
- [ ] Send to stakeholders for review (Katherine)
- [ ] Copyedit (Megan)
- [ ] Publish (Megan)

## Acceptance Criteria
- [ ] Revised page is live on the site

**Comment by bethpottsVADEPO** [comment]
@katherine-fung 
1/ I like the way you rephrased, and you're right that we'll just need to double check that we haven't changed the meaning.
2/ Since the lead-in to the bullets is "**you** must meet...." I'd rephrase the first bullet to this--and I sort of mashed up your 2 approaches:
- You're requesting a Board Appeal within 1 year of our decision on your initial claim, Supplemental Claim, or Higher-Level Review (the 1-year timeframe starts from the date on your decision letter)

Then because a big focus for this work is to clarify how to go about choosing the right DR option, let's get a link in there after bullets to our "Choosing" article. Play around with it, but I could see a couple ways to go about this:
 1/ Bring the text link "Learn about contested claims" up to run in with the third bullet. Add a text link that's not nested under the bullets, but that is relevant to all the text in the blue box (in other words, flush left): Find out more about choosing a decision review option
 2/ Possibly an H3: 
**H3: Other decision review options**
Plain text: Be sure you're choosing the decision review option that's right for you. [possibly a sentence with a nod to what the other 2 options are called--"You may have the option to request a higher-level review or a Supplemental Claim"]
Text link: Learn more about how to choose a decision review option

Something I'd like to talk with our Board Appeal POC about is whether we can find another way to say "Board Appeal options" since we're already using the term "option" for the 3 decision review options. So for that H2 under the blue box, could we instead say something like this for the H2?: What are the Board Appeal choices? / How do I choose the type of Board Appeal I want? (I don't love this wording, but you get the picture--how can we avoid "options" again.) Then, for the H3s, just get rid of the "Option 1/2/3." 

I was hoping you might do a little research on the Board Appeal choices: direct review, new evidence, and request a hearing. See if there's a precedent out there for referring to these in any other way but as "options." Just so we have done our due diligence before meeting with stakeholders about this.

**Comment by katherine-fung** [comment]
Hi @bethpottsVADEPO, 

2/2 - I rephrased the first bullet. And added a link for "Learn more about other decision review options." 

As we discussed, I refer to the different Board Appeal options as "types" on the page. I added a bulleted list for the types (direct review, evidence submission, and Board Hearing). 

http://preview-prod.vfs.va.gov/preview?nodeId=3029

Who are the stakeholders for Board Appeals?

**Comment by bethpottsVADEPO** [comment]
@katherine-fung I think I figured out a way for us to answer the question (which I rewrote) and introduce the H3s without the bulleted list of types of Board Appeals. I made some other edits as well. Want to take a look and let me know if you think it's ready to share with SMEs? Thanks!

**Comment by katherine-fung** [comment]
Thank you @bethpottsVADEPO! The edits work really nicely. I'll message you about sending this to stakeholders.

**Comment by bethpottsVADEPO** [comment]
@katherine-fung Want to put an update here on where we are with this work? Thanks!

**Comment by katherine-fung** [comment]
Hi @bethpottsVADEPO. I followed up with John Gosnell about reviewing this page on 8/2 and 8/16. I can follow up with him again today by email.

**Comment by katherine-fung** [comment]
Good morning @bethpotts. Thanks for working with me to incorporate the majority of the SMEs' feedback on the Board Appeals edits! I've made a few changes to the "How do I request a Board Appeal?" and "What happens next?" sections to incorporate the rest of their feedback. 

Here's the Drupal preview: 
https://preview-prod.vfs.va.gov/preview?nodeId=3029 

I think we had said we would do some more thinking about: 
1. the question in the featured content (currently "Is a Board Appeal an option for me?" and
2. the phrasing of "The Board's goal is to complete an Evidence Submission within 550 days (1.5 years)." 
(Does this work any better? "The Board's goal is to make a decision on an Evidence Submission within 550 days (1.5 years).")

**Comment by bethpottsVADEPO** [comment]
I really like the way you incorporated stakeholder feedback @katherine-fung! I think we should see how "Is a Board Appeal an option for me?" goes over with the next round of reviewers at the Board (the front office). I think it works. 

Take a look at my edits and please let me know if you think the way I messaged things sounds right, especially about the letter telling the Veteran that their appeal has been added to the docket. Thanks!!

**Comment by katherine-fung** [comment]
Thanks, @bethpottsVADEPO! I think the changes work well. 

I'll Slack you and @aprocik1 tomorrow about sending the updated preview back to John and his team for review.

**Comment by katherine-fung** [comment]
I sent the updated preview to SMEs on Wednesday, 1/17. Waiting to hear back. 

Also copying and pasting notes from Beth about future work that's pending until we hear back from SMEs about the Board Appeals page: 

> - We need to add a link from the SC page to the Choosing page in a similar way that we do from the Board Appeals page. (We can workshop this sometime. I have some ideas. We could possibly move that "new and relevant evidence" content out of the blue box and jump link to it on the same page. That would free up space for an "Other decision review options" H3 there instead.)
> 
> - I'm wondering if we adjust that H2 in the blue box on the SC page to match the direction with Board Appeals (pulling back from "the right decision review option" to just "an option." We'd probably want to get agreement from Shireen et al if we want to do that. And we also wouldn't do it if general council says no to that wording for the Board Appeals page.
> 
> - Once all these updates are published to the 2 pages, Julie would love for you to go show the product team how we've improved these pages. But timing-wise, I think it makes sense to wait until we have all the updates in production.

**Comment by katherine-fung** [comment]
I received feedback from the SME, and posted a few additional questions for them in the comments of the doc. Waiting to hear back. 

https://dvagov-my.sharepoint.com/:w:/g/personal/katherine_fung_va_gov/EZeX3MGNy01Ms0CPGLdpqIQBINPD03aYg1AQmcxLoC2AcQ?email=Anthony.Procik%40va.gov&e=lPUfqF

**Comment by katherine-fung** [comment]
Hi @aprocik1, 

I've resolved the comments the Office of the Chief Counsel had for the draft of the Board Appeals page. Can you review draft before we pass to copy edit? 

Here's the inline comparison of the latest draft and previous version: 
https://prod.cms.va.gov/node/3029/revisions/view/837343/843704/visual_inline

In summary, the SME feedback was: 
1. In the intro, introduce the shorthand "the Board" to mean the "Board of Veterans Appeals."
2. In the blue featured content box, re-phrase the first bullet point for accuracy.
3. In the section "Are there different types of Board Appeals?" we can't refer to each type as "a Direct Review," "an Evidence Submssion," or "a Hearing" with an article before each type.  
4. In the section "What happens next?" remove the sentence about VA scheduling exams for the Veteran because that doesn't happen as part of a Board Appeal. 

Here's the Word doc with their comments: 
https://dvagov-my.sharepoint.com/:w:/g/personal/katherine_fung_va_gov/EZeX3MGNy01Ms0CPGLdpqIQBC-JyOp6VrnKnY8fcXDc0WQ?e=l0Roi4

cc @bethpottsVADEPO for awareness

**Comment by aprocik1** [comment]
@katherine-fung, thanks for making these edits. I appreciate how you incorporated the SME's feedback. I think this is a big improvement from the currently published page.

@megzehn, this page is ready for you to copyedit and publish. Thanks!

@bethpottsVADEPO, I've cc'd you for awareness.

**Comment by bethpottsVADEPO** [comment]
@katherine-fung I don't understand this bullet: The most recent decision on your claim was an initial claim, Supplemental Claim, or a Higher-Level Review decision

**Comment by katherine-fung** [comment]
I'm confirming that @bethpottsVADEPO and the SME worked out that last bullet. cc @aprocik1 

This page is ready for copy edit now, @megzehn. We changed a bunch of things based on feedback from the SME/lawyer. Let me know if it's easier to talk through anything for context. 

http://preview-prod.vfs.va.gov/preview?nodeId=3029 

http://preview-prod.vfs.va.gov/preview?nodeId=3029

**Comment by megzehn** [comment]
@katherine-fung and @aprocik1, any thoughts on how to break up this paragraph in the blue box?

> A Board Appeal may be an option for you if you submit your request within 1 year of the decision on your initial claim, Supplemental Claim, or Higher-Level Review (the 1-year timeframe starts from the date on your decision letter).

Would this work?

> A Board Appeal may be an option for you if you submit your request within 1 year of the date on your decision letter. This could be the decision letter from an initial claim, Supplemental Claim, or Higher-Level Review.

Aside from that sentence, I have no other edits! I copy edited and went through the prepub checklist, so it's ready to go. Just double confirming that we're OK to publish?

**Comment by megzehn** [comment]
Never mind! Katherine and I talked and we're going to leave what's there. 

The only change I made was adding "Option 1, Option 2..." under ways to request a Board Appeal to [follow our updated accessibility standards](https://design.va.gov/content-style-guide/page-titles-and-section-titles).

**Comment by katherine-fung** [comment]
Closing this ticket
